mmdet/models/dense_heads/yolact_head.py

```diff
@@ -0,0 +1,931 @@
+            self.loss_bbox_weight = loss_bbox.get('loss_weight', 1.0)
+            sampler_cfg = dict(type='PseudoSampler')
+            self.sampler = build_sampler(sampler_cfg, context=self)
+            self.sampling = False
+
+    def _init_layers(self):
+        """Initialize layers of the head."""
+        self.relu = nn.ReLU(inplace=True)
+        self.head_convs = nn.ModuleList()
+        for i in range(self.num_head_convs):
+            chn = self.in_channels if i == 0 else self.feat_channels
+            self.head_convs.append(
+                ConvModule(
+                    chn,
+                    self.feat_channels,
+                    3,
+                    stride=1,
+                    padding=1,
+                    conv_cfg=self.conv_cfg,
+                    norm_cfg=self.norm_cfg))
+        self.yolact_cls = nn.Conv2d(
+            self.feat_channels,
+            self.num_anchors * self.cls_out_channels,
+            3,
+            padding=1)
+        self.yolact_reg = nn.Conv2d(
+            self.feat_channels, self.num_anchors * 4, 3, padding=1)
+        self.yolact_coeff = nn.Conv2d(
+            self.feat_channels,
+            self.num_anchors * self.num_protos,
+            3,
+            padding=1)
+
+    def init_weights(self):
+        """Initialize weights of the head."""
+        for m in self.head_convs:
+            xavier_init(m.conv, distribution='uniform', bias=0)
+        xavier_init(self.yolact_cls, distribution='uniform', bias=0)
+        xavier_init(self.yolact_reg, distribution='uniform', bias=0)
+        xavier_init(self.yolact_coeff, distribution='uniform', bias=0)
+
+    def forward_single(self, x):
+        """Forward feature of a single scale level.
+
+        Args:
+            x (Tensor): Features of a single scale level.
+
+        Returns:
+            tuple:
+                cls_score (Tensor): Cls scores for a single scale level \
+                    the channels number is num_anchors * num_classes.
+                bbox_pred (Tensor): Box energies / deltas for a single scale \
+                    level, the channels number is num_anchors * 4.
+                coeff_pred (Tensor): Mask coefficients for a single scale \
+                    level, the channels number is num_anchors * num_protos.
+        """
+        for head_conv in self.head_convs:
+            x = head_conv(x)
+        cls_score = self.yolact_cls(x)
+        bbox_pred = self.yolact_reg(x)
+        coeff_pred = self.yolact_coeff(x).tanh()
+        return cls_score, bbox_pred, coeff_pred
+
+    @force_fp32(apply_to=('cls_scores', 'bbox_preds'))
+    def loss(self,
+             cls_scores,
+             bbox_preds,
+             gt_bboxes,
+             gt_labels,
+             img_metas,
+             gt_bboxes_ignore=None):
+        """A combination of the func:`AnchorHead.loss` and func:`SSDHead.loss`.
+
+        When self.use_OHEM == True, it functions like `SSDHead.loss`,
```

Comment:
The rst syntax uses double \`: When \`\`self.use_OHEM == True\`\`, it functions like \`\`SSDHead.loss\`\`